server/src/main/kotlin/org/javacs/kt/KotlinTextDocumentService.kt

```diff
@@ -263,6 +266,7 @@ class KotlinTextDocumentService(
     fun lintAll() {
         debounceLint.submitImmediately {
             sp.compileAllFiles()
+            sp.saveAllFiles()
```

Comment:
Does separately emitting compilation outputs have a non-negligible performance impact?

Reply:
Well, the tests I've done were fine, but they were not exhaustive. With that said, this particular part of the code is done async, so I don't think it should be too problematic. I suggest you check it yourself just in case.

Besides this, the only other time we emit code is on file save (which doesn't happen that often, and only for one file at once).